### ISSUE TITLE
Create a Getting Started page in documentation

### ISSUE THREAD

**Issue by lphk92** [phosphorjs/phosphor#329]
I'm completely new to the project, so having a Getting Started page or at least some more info in the README would be really useful.

**Comment by PerBothner** [comment]
In particular, I can't find any instructions for building or bootstrapping the source code, nor can I find instructions for downloading pre-built packages.

Another problem: The home page describes PhosphorJS as a JavaScript library, but all the source and all the examples appear to be in TypeScript. Which is fine (TypeScript seems pretty cool), but it would be nice to have some examples using plain JavaScript.

**Comment by DanielSokil** [comment]
Getting started example would be great!

**Comment by abalter** [comment]
There are [these examples](https://phosphorjs.github.io/examples.html) but they are not directly linked to on the home page.

**Comment by PerBothner** [comment]
Ah yes, I remember seeing that.  It is still missing "getting started" type instructions (a la "hello world").  I could cobble something together using the `bundle.js` file, but that basically entails reverse engineering things.

It would also be nice to have a smaller bundle.

For now I'm happy with my [fork of nwjs-menu-browser](https://github.com/PerBothner/nwjs-menu-browser), though it is still missing keyboard navigation.  It is less than 1/10 the size of `bundle.js`.  Once PhosphorJS's DockPanel functionality is comparable or better than GoldenLayout (which supports dragging to a tab or popping out to new window) then I'd be interested in switching.

**Comment by abalter** [comment]
I totally agree. There are no instructions for getting started. I'm a good javascript programmer, but I don't know how to use build tools. Also, the size of `bundle.js` in different examples is different. Must have something to do with how the example is built?

Phosphor has been chosen by the Eclipse group for their new cloud/desktop IDE Theia. I think it may get more play than GL. But there is no way to get started.

**Comment by Skorkmaz88** [comment]
> There are [these examples](https://phosphorjs.github.io/examples.html) but they are not directly linked to on the home page.

They are old, and no longer apply to current version
